Makefile

```diff
@@ -143,6 +143,10 @@ else ifeq ($(BUILDX),true)
 endif
 
 build-base:
+	docker pull centos:8
+	docker tag centos:8 centos:7
+	docker pull debian:9
+	docker tag debian:9 debian:8
```

Comment:
This is a little scary and seems misleading; since the toolchain versions are different in CentOS 7<->8 and Debian 8<->9 we are more than likely going to create incompatibility when installing packages built on newer versions of a distro and installed on older distros. Has this been tested with package installs?

Reply:
yup the CI tests it.

As discussed at stnadup openssl is the reason we can't use a single debian binary for all debian distributions but centos 7/8 are compatible as well as debian 8/9. This gives us the advantage of using non-EOL base distributions so there's less hax's while still being able to build / backport Kong to them